x-pack/plugins/file_upload/public/components/import_complete_view.tsx

```diff
@@ -90,69 +96,131 @@ export class ImportCompleteView extends Component<Props, {}> {
   }
 
   _getStatusMsg() {
+    if (this.props.failedPermissionCheck) {
+      return (
+        <EuiCallOut
+          title={i18n.translate('xpack.fileUpload.importComplete.uploadFailureTitle', {
+            defaultMessage: 'Unable to upload file',
+          })}
+          color="danger"
+          iconType="alert"
+        >
+          <p>
+            {i18n.translate('xpack.fileUpload.importComplete.permissionFailureMsg', {
+              defaultMessage:
+                'You do not have permission to create or import data into index "{indexName}".',
+              values: { indexName: this.props.indexName },
+            })}
+          </p>
+          <EuiLink
+            href={getDocLinks().links.maps.importGeospatialPrivileges}
+            target="_blank"
+            external
+          >
+            {i18n.translate('xpack.fileUpload.importComplete.permission.docLink', {
+              defaultMessage: 'View file import permissions',
+            })}
+          </EuiLink>
+        </EuiCallOut>
+      );
+    }
+
     if (!this.props.importResults || !this.props.importResults.success) {
-      return i18n.translate('xpack.fileUpload.uploadFailureMsg', {
-        defaultMessage: 'File upload failed.',
-      });
+      const errorMsg =
+        this.props.importResults && this.props.importResults.error
+          ? i18n.translate('xpack.fileUpload.importComplete.uploadFailureMsgErrorBlock', {
+              defaultMessage: 'Error: {reason}',
+              values: { reason: this.props.importResults.error.error.reason },
+            })
+          : '';
+      return (
+        <EuiCallOut
+          title={i18n.translate('xpack.fileUpload.importComplete.uploadFailureTitle', {
+            defaultMessage: 'Unable to upload file',
+          })}
+          color="danger"
+          iconType="alert"
+        >
+          <p>
+            {i18n.translate('xpack.fileUpload.importComplete.uploadFailureMsg', {
+              defaultMessage: 'Unable to upload file. {errorMsg}',
```

Comment:
As the title is "Unable to upload file", can we remove that text here to avoid the repetition?